CONTRIBUTING.md

```diff
@@ -14,8 +14,6 @@ This page provides information about contributing code to the Jenkins core codeb
    - Apache Maven 3.8.1 or above. You can [download Maven here](https://maven.apache.org/download.cgi).
      In the Jenkins project we usually use the most recent Maven release.
    - Any IDE which supports importing Maven projects.
-   - Install [Node.js 20.x](https://nodejs.org/en/). **Note:** only needed to work on the frontend assets found in the `war` module.
```

Comment:
Our build system installs Node and Yarn, so these instructions should instruct people to use it rather than installing a separate (and possibly incompatible) copy of Node and Yarn outside of our build system.